src/main/java/mate/academy/dao/impl/OrderDaoImpl.java

```diff
@@ -0,0 +1,74 @@
+package mate.academy.dao.impl;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.OrderDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Order;
+import mate.academy.model.User;
+import mate.academy.util.HibernateUtil;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import org.hibernate.query.Query;
+
+@Dao
+public class OrderDaoImpl implements OrderDao {
+    @Override
+    public Optional<List<Order>> getByUser(User user) {
```

Comment:
Optional of list - this looks like new approach))
```suggestion
    public List<Order> getByUser(User user) {
```